styleguide/derek/pattern-components/images/image.ejs

```diff
@@ -0,0 +1 @@
+<img src="http://placekitten.com/300/400"/>
```

Comment:
They didnt have the right aspect ration available in smaller size but Ill try to get closer.